lib/screens_web/controllers/v2/screen_controller.ex

```diff
@@ -227,4 +226,7 @@ defmodule ScreensWeb.V2.ScreenController do
   # While app IDs in configuration still have the "_v2" suffix, but this suffix has been removed
   # from JS/CSS entrypoints, we temporarily need to translate from one to the other.
   defp strip_v2(app_id), do: app_id |> to_string() |> String.replace("_v2", "")
+
+  defp real_screen?(%{params: %{"is_real_screen" => "true"}}), do: true
+  defp real_screen?(_), do: false
```

Comment:
Perhaps we could locate this check in `LogScreenData` rather than duplicating the parameter name across these two controllers? i.e. instead of deriving an `is_screen` variable from the `params` and then passing that into the logging functions, pass in `params` itself and the logging module can take care of making this decision (in a single shared private function).